ts/components/dialogs/connect_wallet_dialog.tsx

```diff
@@ -0,0 +1,335 @@
+import { DialogContent, DialogOverlay } from '@reach/dialog';
+import '@reach/dialog/styles.css';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { Button } from 'ts/components/button';
+import { Icon } from 'ts/components/icon';
+import { Heading, Paragraph } from 'ts/components/text';
+import { colors } from 'ts/style/colors';
+import { zIndex } from 'ts/style/z_index';
+// TODO(kimpers): New providers needed!
+import { Providers } from 'ts/types';
+import { utils } from 'ts/utils/utils';
+
+const MainHeading = styled(Heading).attrs({
+    asElement: 'h3',
+    marginBottom: '0',
+})`
+    font-size: 34px !important;
+    line-height: 42px !important;
+    display: inline-block;
+
+    @media (max-width: 768px) {
+        font-size: 28px !important;
+        line-height: 38px !important;
+    }
+`;
+
+const StyledDialogOverlay = styled(DialogOverlay)`
+    background: none !important;
+    z-index: ${zIndex.overlay};
+
+    @media (max-width: 768px) {
+        background: white !important;
+    }
+`;
+
+const StyledDialogContent = styled(DialogContent)`
+    width: 571px !important;
+    background: ${props => props.theme.bgColor};
+    border: 1px solid #e5e5e5;
+
+    @media (max-width: 768px) {
+        height: 100vh !important;
+        width: 100vw !important;
+        margin: 0 !important;
+        padding: 30px !important;
+
+        border: none;
+    }
+`;
```

Comment:
What is going on here with all the `!important`?

Reply:
`@reach/dialog` sets some specific styling styling, figured it was ok to override with `!important` since it's isolated to the component but I realize now that their docs have info about how to properly override their default styles so I will fix tomorrow